migrations/versions/f87fee85dc66_create_e_case_insensitive_discount_codes.py

```diff
@@ -0,0 +1,31 @@
+"""Create e case insensitive discount codes
+
+Revision ID: f87fee85dc66
+Revises: 6e5c574cbfb8
+Create Date: 2019-04-02 02:22:08.086011
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'f87fee85dc66'
+down_revision = '6e5c574cbfb8'
+
+def upgrade():
+    op.execute("UPDATE discount_codes SET code = concat(code, '_') where code not in (SELECT DISTINCT ON (upper(code)) code FROM discount_codes);",
```

Comment:
Will have to verify this query

Reply:
@iamareebjamal is the query okay?

Reply:
Thanks for reminding. I'll check today

Reply:
There's an issue in this, this will rename all duplicate accounts. We don't want that. We want duplicates to not happen inside same event, global duplicates may happen. So you have to group by event as well